evalml/pipelines/components/transformers/preprocessing/text_featurizer.py

```diff
@@ -22,12 +23,9 @@ def __init__(self, text_columns=None, random_state=0, **kwargs):
             random_state (int, np.random.RandomState): Seed for the random number generator.
 
         """
-        self._ft = import_or_raise("featuretools", error_msg="Package featuretools is not installed. Please install using `pip install featuretools[nlp_primitives].`")
-        self._nlp_primitives = import_or_raise("nlp_primitives", error_msg="Package nlp_primitives is not installed. Please install using `pip install featuretools[nlp_primitives].`")
-        self._trans = [self._nlp_primitives.DiversityScore,
-                       self._nlp_primitives.MeanCharactersPerWord,
-                       self._nlp_primitives.PartOfSpeechCount,
```

Comment:
We removed `PartOfSpeechCount` because it was slow, right?

Reply:
Yep! It took significantly longer than any of the other primitives without adding any noticeable performance.